spec/dummy/client/webpack.common.config.js

```diff
@@ -19,8 +17,7 @@ const assetLoaderRules = [
     use: {
       loader: 'file-loader',
       options: {
-        // Leading slash is 100% needed
-        name: '/images/[name]-[hash].[ext]',
+        name: 'images/[name]-[hash].[ext]',
```

Comment:
We removed the leading slash because we already appended it as a trailing slash in webpack.client.rails.build.config.js as explained.